main.py

```diff
@@ -128,6 +125,7 @@ def execute_abcd_assessment_for_videos(config: Configuration):
     brand_assessment.get("video_assessments").append(video_assessment)
 
     if config.bq_table_name:
+      bq_service = BigQueryService(config.project_id)
```

Comment:
Move this into store_in_bq, leverage config to pass parameter.
Exposing the internals of store_in_bq adds BigQueryService dependency here. Anyone modifying the code for AWS for example will have to remove 2 dependencies instead of 1.

Reply:
`store_in_bq` would still be a dependency. Moving this into `store_in_bq` just shifts the dependency down one level. I recommend leaving this as-is until the need to refactor comes up. [Premature optimization is the root of all evil](https://m.xkcd.com/1691/).

Reply:
+1 this is still a dependency. If this gets removed, or replaced, it would be the whole condition.